### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by anirudhsai20**
## Related Issue
Readme file enhancement

Closes: #68 

## Describe the changes you've made
I have added a list of all the Open Source programs in which this project has been listed in. 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

dcli

### CODE REVIEW

**Review by honeybhardwaj — APPROVED** [review]
LGTM!!

File: README.md [hunk 1]
```diff
@@ -98,6 +98,13 @@ git push
 
 **9.** **Congratulations!** Sit and relax, you've made your contribution to Dynamic-CLI project.
 
+## Dynamic CLI is a part of these open source programs:
```
Comment (Siddhant-K-code) [inline README.md:101]: Don’t add a trailing `:` to headings

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

**Comment by GouravSardana** [comment]
@anirudhsai20 Don't you think this image is really very big for ReadME?

**Comment by anirudhsai20** [comment]
Ok. I will reduce the size of the image.

**Comment by anirudhsai20** [comment]
I have reduced the size of the image. Is this fine or do you want it to be smaller?

**Comment by GouravSardana** [comment]
@anirudhsai20 Can you please fix this one check. So that I can merge

**Comment by anirudhsai20** [comment]
I hope the issue is fixed now.

**Pull request merged**